src/main/java/net/sf/mzmine/modules/visualization/spectra/simplespectra/spectraidentification/spectraldatabase/SpectraIdentificationSpectralDatabaseModule.java

```diff
@@ -67,10 +67,12 @@ public ExitCode runModule(@Nonnull MZmineProject project, @Nonnull ParameterSet
   public static void showSpectraIdentificationDialog(final Scan scan,
       final SpectraPlot spectraPlot) {
 
-    final ParameterSet parameters = new SpectraIdentificationSpectralDatabaseParameters();
+    final SpectraIdentificationSpectralDatabaseParameters parameters =
```

Comment:
A comment unrelated to this PR: here you always create a new parameter set instance with default values. It is better to use ```MZmineCore.getConfiguration().getModuleParameters(SpectraIdentificationSpectralDatabaseModule.class);```

Reply:
That way the parameters will be remembered and saved on MZmine exit

Reply:
Where do I have to add the module / parameters to get them into the configuration?

We dont want the module to be listed in the menu - therefore not in the ModuleList correct?

Reply:
The parameter set instances are created automagically during MZmine startup.

Please add the module to MZmineModulesList. Menu items are only created for those modules that implement MZmineProcessingModule.

Reply:
done.
@Annexhc you might want to do something similar for all the other spectra identification modules.
Change to RunnableModule, add to module list, retrieve Parameters from configuration.
see commit 9b39a56

Reply:
@Annexhc Yes please

Reply:
I didnt know that. Will do it!